apps/web/src/config/constants/meta.ts

```diff
@@ -1,49 +1,63 @@
 import memoize from 'lodash/memoize'
 import { ContextApi } from '@pancakeswap/localization'
 import { PageMeta } from './types'
+import { ASSET_CDN } from './endpoints'
 
 export const DEFAULT_META: PageMeta = {
   title: 'PancakeSwap',
   description:
     'The most popular AMM on BSC by user count! Earn CAKE through yield farming or win it in the Lottery, then stake it in Syrup Pools to earn more tokens! Initial Farm Offerings (new token launch model pioneered by PancakeSwap), NFTs, and more, on a platform you can trust.',
-  image: 'https://pancakeswap.finance/images/hero.png',
+  image: `${ASSET_CDN}/web/og/hero.jpg`,
 }
 
 interface PathList {
-  paths: { [path: string]: { title: string; basePath?: boolean; description?: string } }
+  paths: { [path: string]: { title: string; basePath?: boolean; description?: string; image?: string } }
   defaultTitleSuffix: string
 }
 
 const getPathList = (t: ContextApi['t']): PathList => {
   return {
     paths: {
       '/': { title: t('Home') },
-      '/swap': { basePath: true, title: t('Exchange') },
-      '/limit-orders': { basePath: true, title: t('Limit Orders') },
-      '/add': { basePath: true, title: t('Add Liquidity') },
-      '/remove': { basePath: true, title: t('Remove Liquidity') },
-      '/liquidity': { title: t('Liquidity') },
+      '/swap': { basePath: true, title: t('Exchange'), image: `${ASSET_CDN}/web/og/swap.jpg` },
+      '/limit-orders': { basePath: true, title: t('Limit Orders'), image: `${ASSET_CDN}/web/og/limit.jpg` },
+      '/add': { basePath: true, title: t('Add Liquidity'), image: `${ASSET_CDN}/web/og/liquidity.jpg` },
+      '/remove': { basePath: true, title: t('Remove Liquidity'), image: `${ASSET_CDN}/web/og/liquidity.jpg` },
+      '/liquidity': { title: t('Liquidity'), image: `${ASSET_CDN}/web/og/liquidity.jpg` },
       '/find': { title: t('Import Pool') },
       '/competition': { title: t('Trading Battle') },
```

Comment:
Should we add a image for TRADING COMPETITION ?

Reply:
Each competition should have its own image. Since all of them are closed, I think there's no real benefits to add it now